web-ui/public/index.html

```diff
@@ -25,8 +25,8 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.15.4/video-js.min.css" media="print" onload="this.media='all'">
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/video.js/7.15.4/video.min.js"></script>
```

Comment:
Do we also want to update videojs, leave as is or match the sample code snippet from [the IVS documentation](https://docs.aws.amazon.com/ivs/latest/userguide/player-videojs.html)?

Reply:
Updated here: 1aab15c